docs/Reference/Rest_API/Rest.md

```diff
@@ -4,16 +4,22 @@ description: Teku RESTful API
 
 # REST API
 
-[Enable the REST API service](#enable-the-rest-api-service) from the command line, and enable
-the API documentation endpoint to interact with APIs using Swagger UI.
+[Enable the REST API service](#enable-the-rest-api-service) from the command line. You can also
+interact with APIs using Swagger UI by enabling the API documentation endpoint.
 
-[Teku REST API documentation] is also available.
+View the [REST API documentation] for more information about the available APIs.
 
 ## Enable the REST API service
 
 Enable the REST API service by setting the [`--rest-api-enabled`](../CLI/CLI-Syntax.md#rest-api-enabled)
 command line option to `true`.
 
+!!! warning
+
+    Only trusted parties should access the REST API. Do not directly expose these APIs publicly on
+    production nodes. Use [`--rest-api-host-allowlist`](../CLI/CLI-Syntax.md#rest-api-host-allowlist)
+    to limit access to trusted parties.
```

Comment:
I would cut this suggestion about the `allowlist` as this is one way to limit access, but isn't necessarily sufficient:
```suggestion
    Only trusted parties should access the REST API. Do not directly expose these APIs publicly on
    production nodes.
```